scripts/versions

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 # Versions to be used
 declare -A VERSIONS=(
     ["cni-plugins"]=v0.8.7
-    ["conmon"]=v2.0.26
+    ["conmon"]=master
```

Comment:
This version will be used for the static binary bundle, too. So it will not work for that case.

Reply:
alas